projects/ngx-openlayers/src/lib/view.component.ts

```diff
@@ -42,12 +42,19 @@ export class ViewComponent implements OnInit, OnChanges, OnDestroy {
   @Input()
   zoomAnimation = false;
 
-  constructor(private host: MapComponent) {}
+  @Output()
+  onZoomChange: EventEmitter<ObjectEvent>;
+
+  constructor(private host: MapComponent) {
+    this.onZoomChange = new EventEmitter<ObjectEvent>();
+  }
 
   ngOnInit() {
     // console.log('creating ol.View instance with: ', this);
     this.instance = new View(this);
     this.host.instance.setView(this.instance);
+
+    this.instance.on('change:zoom', (event: ObjectEvent) => this.onZoomChange.emit(event));
```

Comment:
I'm wondering if we should choose a convention between `@Output` name property and `Event` string name to simplify dev experience.

Do we have others like this in the lib?

Here we have `onZoomChange` => `change:zoom`. Words are interverted. Seems a bit hard to find when we are coming from the OL API. 

Could we discuss of this ?
/cc @Yakoust & others

Reply:
@davinkevin, we have this everywhere. 
BTW, we also suffix 'on' for `@outputs`. 
For me, word inversion is ok, but, we should have `zoomChanged` instead of `onZoomChange`

Here again, we should open an issue to change this everywhere

Reply:
After a discussion with @davinkevin We think that the best is to have term not inverted and `on` prefix kept. So having `onZoomChange`.

This change should be made for this property on this PR, and an issue should be opened to homogenize this in all the library

Reply:
@Yakoust do you mean `onChangeZoom`?

Reply:
yes @aymeric-duchein sorry `onChangeZoom`

Reply:
done!